### PR TITLE
Feature/63/basic task management

### LINKED ISSUE #63: Basic task management
- Task class
- Stealing task
- Init task at start of game
- Show tasks in ui

### PR DESCRIPTION
**Pull request by JElgar**
closes #63

### CODE REVIEW

File: Assets/Scripts/Environment/TaskSetter.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
+using System.Collections;
```
Comment (JElgar) [inline Assets/Scripts/Environment/TaskSetter.cs:1]: Needs comments

File: Assets/Scripts/UI/TaskUI.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    public GameObject tasksList;
+
+    private Task[] tasks;
+
+    void Start() {
+      tasks = new Task[10];
+      Debug.Log("Starting");
+      Task task = new Task();
+      task.description = "Testeroo";
+      tasks[0] = task;
```
Comment (JElgar) [inline Assets/Scripts/UI/TaskUI.cs:18]: Nono

File: Assets/Scripts/Utils/GameManager.cs [hunk 3]
```diff
@@ -4,23 +4,38 @@
...
     public GameObject seekerPrefab;
+
+    // Current instance of the GameManager singleton
     public static GameManager instance;
-    
-    public enum Team
-    {
+  
+    // The teams
+    // TODO Do we use this?
```
Comment (JElgar) [inline Assets/Scripts/Utils/GameManager.cs:25]:
```suggestion
    // TODO We should use this more
```

File: Assets/Scripts/Utils/GameManager.cs [hunk 4]
```diff
@@ -30,19 +45,15 @@ public enum Team
       }
     }
 
+    // Handle robber being captured
+    // TODO move this into capturable
     public void OnRobberCapture(GameObject robber) {
       PhotonView view = robber.GetComponent<PhotonView>();
       GameObject jail = GameObject.Find("/Jail/JailSpawn");
       NetworkManager.instance.SetPlayerProperty("Captured", true, view.Owner);
       view.RPC("MovePlayer", view.Owner, jail.transform.position);
```
Comment (JElgar) [inline Assets/Scripts/Utils/GameManager.cs:54]: Lets move this

File: Assets/Scripts/Utils/JsonListHelper.cs [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        Wrapper<T> wrapper = new Wrapper<T>();
+        wrapper.Items = array;
+        return JsonUtility.ToJson(wrapper, prettyPrint);
+    }
+
+    [System.Serializable]
+    private class Wrapper<T> {
+        public T[] Items;
+    }
+}
```
Comment (JElgar) [inline Assets/Scripts/Utils/JsonListHelper.cs:27]: This can go

File: Assets/Scripts/Utils/GameManager.cs (hunk 3 above)
Comment (JElgar) [reply Assets/Scripts/Utils/GameManager.cs:25]: This should go its now in its own file

**Review by JDBryan — APPROVED** [review]
looks good to me

**Pull request merged**